core/spying/type-matcher.ts

```diff
@@ -1,9 +1,9 @@
 import { ArgumentStringifier } from "../stringification";
 
-type Tester = {
-   test: (value: any) => boolean;
-   stringify: () => string;
-};
+interface ITester {
```

Comment:
could we export this definition out of the file (I know it was like that already :) ) Unless you think there's a good reason to pair the definition?